test.sh

```diff
@@ -55,10 +56,18 @@ function filename2imagename {
 }
 
 # Run a test in a docker container
-# Usage: run_test_docker <Dockerfile> <env_name>
+# Usage: run_test_docker <Dockerfile> <env_name> <args>...
 function run_test_docker {
-    local dockerfile=$1
+  run_docker /root/influxdb/build.py "${@}" --test --junit-report "--parallel=$PARALLELISM" "--timeout=$TIMEOUT"
+}
+
+# Run a script in a docker container
+# Usage: run_docker <Dockerfile> <env_name> <args>...
```

Comment:
This is no longer correct - I think this would be better though - just have the command be part of `args`.

Also the variable below is representing a docker `command`, not an `entrypoint`.